cmd/proxy/actions/storage.go

```diff
@@ -19,7 +19,7 @@ import (
 // GetStorage returns storage backend based on env configuration
 func GetStorage() (storage.BackendConnector, error) {
 	// changing to mongo storage, memory seems buggy
```

Comment:
nit- maybe get rid of this comment?

Reply:
As per https://github.com/gomods/athens/issues/165 it looks like the issue was fixed!

Reply:
Awesome, so just comment clean up :+1: